packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -101,6 +101,10 @@
 		text-overflow: ellipsis;
 		color: inherit;
 	}
+
+	.dataviews-pagination {
```

Comment:
I've noticed that we use a dataviews internal class to do this. Can we do this differently? In my view, consumers shouldn't target dataviews internal classes, otherwise they'll become public API.

[related thread](https://github.com/WordPress/gutenberg/pull/60284/files#r1546276108)

Reply:
I think the Data Views pagination component cannot be wrapped in an element with a specific class, nor can we add a specific class name.

https://github.com/WordPress/gutenberg/blob/fc78ac9cd55819aae3c0a0ff85a5e273c471acce/packages/dataviews/src/pagination.js#L24-L29

For example, should we add a prop like `paginationClassName` to the `DataViews` component?

https://github.com/WordPress/gutenberg/blob/fc78ac9cd55819aae3c0a0ff85a5e273c471acce/packages/dataviews/src/dataviews.js#L31-L46